README.md

```diff
@@ -0,0 +1,45 @@
+# Source for [rust-for-rustaceans.com](https://rust-for-rustaceans.com)
+
+Home of Reviews and Errata for ["Rust for Rustaceans" by Jon Gjengset](https://nostarch.com/rust-rustaceans).
+
+<kbd><img src="https://github.com/jonhoo/rust-for-rustaceans.com/blob/main/RustforRustaceans_cover.png?raw=true" title="Rust for Rustaceans" alt="book cover art with excited Ferris (Rust mascot)" /></kbd>
+
+# Submit Errata
+
+Found a bug?
+
+Just take a look at [accepted Errata pull requests](https://github.com/jonhoo/rust-for-rustaceans.com/pulls?q=is%3Apr+is%3Aclosed+errata) and submit in similar form.
+
+## Example
+
+Branch / Commit / PR name: "errata print01-ch{nn}-lst{n} syntax error in example"
+
+`````mkdn
+---
+chapter: 7
+page: 103
+kind: code
```

Comment:
```suggestion
kind: code/typo
```

Reply:
Is this to suggest that "code/typo" is a category or that it may be "code" or "typo"?

Reply:
I found the answer:

```bash
rg 'kind:'
```

Updated the README accordingly.